content/docs/instrumenting/exposition_formats.md

```diff
@@ -5,104 +5,96 @@ sort_rank: 6
-human readable, so it is a good fallback candidate (and also understood by
-Prometheus).
-
-### Format variants comparison
-
-|               | Protocol buffer format | Text format |
-|---------------|------------------------|-------------|
-| **Inception** | April 2014 | April 2014  |
-| **Supported in** | Prometheus version `>=0.4.0`, `<2.0.0` | Prometheus version `>=0.4.0` |
-| **Transmission** | HTTP | HTTP |
-| **Encoding** | [32-bit varint-encoded record length-delimited](https://developers.google.com/protocol-buffers/docs/reference/java/com/google/protobuf/AbstractMessageLite#writeDelimitedTo(java.io.OutputStream)) Protocol Buffer messages of type [io.prometheus.client.MetricFamily](https://github.com/prometheus/client_model/blob/086fe7ca28bde6cec2acd5223423c1475a362858/metrics.proto#L76-  L81) | UTF-8, `\n` line endings |
-| **HTTP `Content-Type`** | `application/vnd.google.protobuf; proto=io.prometheus.client.MetricFamily; encoding=delimited` | `text/plain; version=0.0.4` (A missing `version` value will lead to a fall-back to the most recent text format version.) |
-| **Optional HTTP `Content-Encoding`** | `gzip` | `gzip` |
-| **Advantages** | <ul><li>Cross-platform</li><li>Size</li><li>Encoding and decoding costs</li><li>Strict schema</li><li>Supports concatenation and theoretically streaming (only server-side behavior would need to change)</li></ul> | <ul><li>Human-readable</li><li>Easy to assemble, especially for minimalistic cases (no nesting required)</li><li>Readable line by line (with the exception of type hints and docstrings)</li></ul> |
-| **Limitations** | <ul><li>Not human-readable</li></ul> | <ul><li>Verbose</li><li>Types and docstrings not integral part of the syntax, meaning little-to-nonexistent metric contract validation</li><li>Parsing cost</li></ul>|
-| **Supported metric primitives** | <ul><li>Counter</li><li>Gauge</li><li>Histogram</li><li>Summary</li><li>Untyped</li></ul> | <ul><li>Counter</li><li>Gauge</li><li>Histogram</li><li>Summary</li><li>Untyped</li></ul> |
-| **Compatibility** | Version `0.0.3` protocol buffers are also valid version `0.0.4` protocol buffers. | none |
-
-### Protocol buffer format details
-
-Reproducible sorting of the protocol buffer fields in repeated expositions is
-preferred but not required, i.e. do not sort if the computational cost is
-prohibitive.
-
-Each `MetricFamily` within the same exposition must have a unique name. Each
-`Metric` within the same `MetricFamily` must have a unique set of `LabelPair`
-fields. Otherwise, the ingestion behavior is undefined.
+Metrics can be exposed to Prometheus using a simple [text-based](#text-based-format)
+exposition format. There's a variety of [client libraries](/docs/instrumenting/clientlibs/)
+that implement this format for you. If your preferred language doesn't have a client
+library you can [create your own](/docs/instrumenting/writing_clientlibs/).
+
+NOTE: Some earlier versions of Prometheus supported an exposition format based on
```

Comment:
The `NOTE:` here should probably still be `NOTE: **NOTE:**`, as the initial `NOTE:` is translated into a formatting directive that creates a call-out box and is otherwise not displayed, see https://prometheus.io/docs/instrumenting/exposition_formats/

Or was your intention to get rid of the displayed "NOTE" text within that box?

Reply:
Yes, I'm aware of the `NOTE` syntax. In my view the styling makes it unnecessary to include  **NOTE** but I'm not dogmatic about that. Your call.

Reply:
Hmm, I just tried removing it on the live site and it looks a bit odd to me in context without the "NOTE" text:

![note](https://user-images.githubusercontent.com/538008/42063014-08e451b8-7b30-11e8-9a19-3151a7f3e11f.png)

I'd be for keeping it. If we do decide to remove it, we should make that a conscious decision and remove the extra callout words throughout the site, not just in one instance.

Reply:
Okay, I'll make that change. In a future PR I'll adjust the template of admonition blocks so that the admonition type is included automatically.

Reply:
Great idea!